src/gnome-abrt

```diff
@@ -189,6 +100,33 @@ class OopsApplication(Gtk.Application):
         # Disable because of https://bugzilla.redhat.com/show_bug.cgi?id=959811
         gnome_abrt.url.set_async_worker(lambda *args: None)
         self.gnome_settings = None
+        # pylint: disable=W0105
+        '''
+        Again a trick to make xgettext think we are C language and emit the
+        translators comment correctly.
+        See: bugs.launchpad.net/intltool/+bug/377872
+        // Translators: This is a description of --verbose command line option
+        // displayed when a user runs: `gnome-abrt --help'
+        _("Be verbose")
+        '''
+        self.add_main_option('verbose',
+                             ord('v'),
+                             GLib.OptionFlags.NONE,
+                             GLib.OptionArg.NONE,
+                             _('Be verbose'),
+                             None)
+        # pylint: disable=W0105
+        '''
+        // Translators: This is a description of --problem command line option
+        // displayed when a user runs: `gnome-abrt --help'
+        _("Selected problem ID")
+        '''
+        self.add_main_option('problem',
+                             ord('p'),
+                             GLib.OptionFlags.NONE,
+                             GLib.OptionArg.STRING,
+                             _('Be verbose'),
```

Comment:
Too verbose. 

`s/'Be verbose'/'Selected problem ID'/`

Reply:
I don’t know, I feel like we undersell our Verbose Output® technology.